nightlies/cron_wrapper

```diff
@@ -18,6 +18,11 @@ STDOUT="/var/tmp/stdout.$$"
 # $STDOUT and $STDERR are removed when the script exits for any reason
 trap  "rm -f $STDOUT $STDERR" 0
 
+# lock $LOG with file descriptor 200
+# if $LOC is locked by other process - wait for 420 sec
+exec 200>>$LOG
+flock -w 420 200 || exit 1
```

Comment:
1) 420 seems weirdly arbitrary.  Why not, oh, 15 minutes?

2) if you fail to lock the log, you don't want to fail to run the job altogether; just avoid logging, probably, or perhaps fallback to "send email with all output", or "log to a fallback file"

3) you really only need to hold the lock for the time you need to write the lock.